addons/element-remover/addon.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Element remover",
+  "description": "Remove elements inside scratch",
```

Comment:
This is a pretty general addon name. It sounds like it would be used for something with many options. We may want to rename it to something more specific to the features we're offering, or to keep it general-ish, something like "remove distractions".

Reply:
> This is a pretty general addon name. It sounds like it would be used for something with many options. We may want to rename it to something more specific to the features we're offering, or to keep it general-ish, something like "remove distractions".

Good idea. I just did it as a temporary name. I'll rename it after school 😀